apimonitor/monitor/health.py

```
@@ -0,0 +1,162 @@
+                {
+                    'service': service.name,
+                    'url': url,
+                    'error': 'http',
+                    'details': str(e),
+                    'attempt': attempt + 1,
+                    'ts': time.time()
+                }
+            )
+            
+        except Exception as e:
+            last_error = f"Unexpected error: {type(e).__name__}: {str(e)}"
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)